src/veh_interact.cpp

```diff
@@ -2052,20 +2052,18 @@ void veh_interact::display_details( const vpart_info *part )
                        volume_units_abbr() );
     }
 
-    // line 3: (column 1) size,bonus,wheel_width (as applicable)    (column 2) epower (if applicable)
-    if ( part->size > 0 ) {
+    // line 3: (column 1) size, bonus, wheel_width (if applicable)    (column 2) epower (if applicable)
+    if( part->size > 0 ) {
         std::string label;
-        if ( part->has_flag(VPFLAG_CARGO) ) {
-            label = small_mode ? _("Cap") : _("Capacity");
-        } else if ( part->has_flag(VPFLAG_WHEEL) ){
-            label = small_mode ? _("Size") : _("Wheel Size");
-        } else {
-            label = small_mode ? _("Cap") : _("Capacity");
+        if( part->has_flag( VPFLAG_CARGO ) ) {
+            label = small_mode ? _( "Cap" ) : _( "Capacity" );
         }
 
-        fold_and_print(w_details, line+3, col_1, column_width, c_white,
+        if( !label.empty() ) {
```

Comment:
this is useless, entire block can be like that:
```
if( part->size > 0 ) {
   fold_and_print( ... small_mode ? _( "Cap" ) : _( "Capacity" ) ...
}
```

Reply:
No, there are parts that have `size`, but they aren't cargo carriers - like funnel, for example.

Reply:
Okay, if showing of capacity for funnels was a mistake, then condition can be changed to ```size > 0 AND cargo```. Anyway, label variable is useless because it defines only once, and entire code block is overcomplicated.